packages/i18n/messages/de.ts

```diff
@@ -3108,5 +3127,25 @@ Da die KlickerUZH-App noch nicht im iOS-App-Store verfügbar ist, folgen Sie die
       hintLastBlock:
         'Der aktuell laufende Block is der letzte dieses Live Quizzes. Nach Schliessen dieses Blockes kann das Live Quiz beendet werden.',
     },
+    activity: {
+      title: '{type} Aktivitäten',
+      tooltip: 'Aktivitätsprotokoll anzeigen',
+      viewActivityLog: 'Aktivitätsprotokoll anzeigen',
+      element: 'Element',
```

Comment:
after replacing the corresponding translation strings, we should remove the keys again from here

Reply:
done